files.md

```diff
@@ -9,28 +9,62 @@ In order to make the contesting life a little easier, we provide some files you
+   Chinese radios.
 3. [Install Chirp](https://chirp.danplanet.com/projects/chirp/wiki/Download) on your computer.
-4. You are now ready to test [cloning from your radio](https://chirp.danplanet.com/projects/chirp/wiki/Beginners_Guide). If this works, your radio and computer are correctly communicating, and you are ready to write our <a href="assets/programming_files/rr_frequencies.csv" download>CSV file</a>. Dave Casler has [a good video to get you started](https://www.youtube.com/watch?v=Mrpqq-xi00g).
+4. You are now ready to test [cloning from your
+   radio](https://chirp.danplanet.com/projects/chirp/wiki/Beginners_Guide). If
+   this works, your radio and computer are correctly communicating, and you are
+   ready to write our <a href="assets/programming_files/rr_frequencies.csv"
+   download>CSV file</a>. Dave Casler has [a good video to get you
+   started](https://www.youtube.com/watch?v=Mrpqq-xi00g).
 
 For more information, check out [Chirp's Wiki](https://chirp.danplanet.com/projects/chirp/wiki/Home).
 
 
 ## AnyTone D878 instructions
 
-Our <a href="assets/programming_files/d878.zip" download>download for the D878</a> is a zip file containing three files :
+Our <a href="assets/programming_files/d878.zip" download>download for the
+D878</a> is a zip file containing three files :
 - `d878.csv` contains all of the repeater information (frequencies, offsets, tones)
-- `d878-talk-groups.csv` contains a set of talk group definitions for the DMR repeaters
-- `d878-scanlist.csv` contains information that allows you to scan across all repeaters in the Roundabout
+- `d878-talk-groups.csv` contains a set of talk group definitions for the DMR
+  repeaters
+- `d878-scanlist.csv` contains information that allows you to scan across all repeaters in the
+  Roundabout
 
-When you open the CPS software, go to the `Tools > Import` menu and select each of the files you've downloaded in the appropriate category (Channel, Scan List, and/or Talk Groups). Then use the `Program > Write to Radio` menu to send the program to your D878.
+When you open the [CPS
+software](https://support.bridgecomsystems.com/anytone-878-v2-model-cps-firmware-downloads),
+go to the `Tools > Import` menu and select each of the files you've downloaded
+in the appropriate category (Channel, Scan List, and/or Talk Groups). Then use
+the `Program > Write to Radio` menu to send the program to your D878.
+
+## Icom-705 Instructions
+
+*These have been tested with the Icom-705.  If you have an Icom 7300 (or other Icom radio), please
+let us know if this file is also compatible with your radio.*
+
+1. Download and install the [Icom CS-705
+   Software](https://www.icomjapan.com/support/firmware_driver/3067/).
+2. Connect a USB cable from your computer to the Icom 705.
+3. Use the `COM Port > Setting...` command to select the COM port used by the USB connection.
+4. Use the `Clone < Read <- TR` menu to duplicate the Icom's settings to your computer.
```

Comment:
What is the `<` notation, compared with the `>` ? And the `<-` ?

Reply:
It's not a notation - it's the literal text of the menu.

    Read <- TR

Reply:
Oh - but I did get the `>` backwards!